pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeTableDataManager.java

```diff
@@ -240,8 +240,8 @@ protected void doShutdown() {
    * @param ingestionTimeMs Ingestion delay being reported.
    * @param partitionGroupId Partition ID for which delay is being updated.
    */
-  public void updateIngestionDelay(long ingestionTimeMs, int partitionGroupId) {
-    _ingestionDelayTracker.updateIngestionDelay(ingestionTimeMs, partitionGroupId);
+  public void updateIngestionDelay(long ingestionTimeMs, long creationTimeMs, int partitionGroupId) {
+    _ingestionDelayTracker.updateIngestionDelay(ingestionTimeMs, creationTimeMs, partitionGroupId);
```

Comment:
s/creationTimeMs/firstStreamIngestionTimeMs/

Reply:
Addressed in last commit